semantic-conventions/src/opentelemetry/semconv/main.py

```diff
@@ -138,6 +140,14 @@ def add_code_parser(subparsers):
         help="List of key=value pairs separated by comma. These values are fed into the template as is.",
         type=str,
     )
+    parser.add_argument(
+        "--jinja-env-params",
+        "-J",
+        dest="jinja_env_params",
+        action="append",
+        help="List of key=value pairs separated by comma. These values are fed into the jinja2.Environment as is.",
+        type=str,
+    )
```

Comment:
Please mention the new flags in https://github.com/open-telemetry/build-tools/tree/main/semantic-conventions#code-generator (maybe in a separate snippet, if at all, as this is only for more advanced use cases).

Reply:
I added a new section under `#Code generator` for customizing the Jinja Environment. 9f548222a983515fdce52edbad1432bc5ce242f4